### ISSUE TITLE
Green Hills of Stranglethorn

### ISSUE THREAD

**Issue by Raulfin** [dalaranwow/dalaran-wow#4344]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: When pages for the Green HIlls of Stranglethorn sub-quests (Chapter I, Chapter II, Chapter III & Chapter IV) are traded to the player they do not get counted as collected. Placing the pages in the bank vault and then taking them out marks them as collected, and gives the <objective complete> notification. Reloading the toon or the game doesn't work. 

**Expected behaviour**: Traded pages should be marked as collected

**Steps to reproduce the problem**:

1.  Trade a page to a player, the quest will appear to be completed in quest tracker, but no <objective complete> notification.
2. Talk to NPC-Barnil Stonepot in Nesingwarrys camp Stranglethorn Vale
3. The quest will have a gold '?', making it look complete, but the <Continue> button will not usable until the pages are banked and retrieved. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
Don't know the Quest/NPC ID's, wowwiki and wowhead didn't list them.
You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
simply remove the quest from your log and retake it - then you can complete the quest
